File: scrabble-board-manager/src/main/java/com/aryzko/scrabble/scrabbleboardmanager/domain/model/BoardParameters.java
package com.aryzko.scrabble.scrabbleboardmanager.domain.model;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

@Value
@Builder
@Jacksonized
public class BoardParameters {
    private Integer horizontalSize;
    private Integer verticalSize;
    private Integer rackSize;
}

File: scrabble-ui/src/app/clients/board-manager/model/board-parameters.ts
export class BoardParameters {
  public horizontalSize: number;
  public verticalSize: number;
  public rackSize: number;

  constructor(horizontalSize: number, verticalSize: number, rackSize: number) {
    this.horizontalSize = horizontalSize;
    this.verticalSize = verticalSize;
    this.rackSize = rackSize;
  }
}
